### ISSUE TITLE
Save keyboard shortcut doesn't work when focus is in Find pane

### ISSUE THREAD

**Issue by joewiz** [eXist-db/eXide#113]
When the keyboard focus is the Find pane (Edit > Find in Buffer), the keyboard shortcut for Save (File > Save) does not work. To save, you have to escape out of the Find pane, then save. I'm not sure if a workaround is possible with ACE, but if there were, this would be great.
